### DISCUSSION TITLE
Depended hiredis v0.12 has security risk, whether newer versions without security risks can be supported?

### DISCUSSION THREAD

**Discussion by blackholll** [OpenIDC/mod_auth_openidc#1202] [General]
from blackduck , we know hiredis  0.12 has security risk.  but now the lastest mod_auth_openidc only support hiredis 0.12
![WechatIMG345](https://github.com/OpenIDC/mod_auth_openidc/assets/6269605/094f4bd8-a281-4516-9fa2-f8b6d33097df)

**Comment by zandbelt** [comment]
hiredis is merely a (optional) dependency for mod_auth_openidc; it is up to the OS distribution to patch it or upgrade it

**Comment by blackholll** [comment]
I tried install mod_auth_openidc on amzn2 or amzn2023, they all need hiredis.  I think redis is use for store session, since we are use client-cookie.   it there any way to install mod_auth_openidc without hiredis?
![image](https://github.com/OpenIDC/mod_auth_openidc/assets/6269605/0214d2ca-d19d-43d8-91e4-907ddaddd271)

**Reply by zandbelt** [reply]
there's no pre-bulit package for that, you'd have to build the binaries from source yourself

**Reply by blackholll** [reply]
you mean rpm for amzn?  I can build rpm for amzn or install by source code directly,   but when I try  to install mod_auth_openidc,  it says need hiredis 0.12.  

but  hiredis 0.12 has security risk issue， In our company, we are now allowed install package which has security risk issue.

**Reply by blackholll** [reply]
I rebuild the rpm package, and it works.
this is the spec for rpm building:   https://src.fedoraproject.org/rpms/mod_auth_openidc/blob/rawhide/f/mod_auth_openidc.spec
